### PR TITLE
KFLUXINFRA-635: Restore s390x and add a100 node to internal-stage hostconfig

### PR DESCRIPTION
**Pull request by bamachrn**

### CODE REVIEW

**Review by stuartwdouglas — APPROVED** [review]

File: components/multi-platform-controller/staging-downstream/host-config.yaml [hunk 1]
```diff
@@ -89,3 +89,9 @@ data:
   host.ppc1.user: "root"
   host.ppc1.secret: "internal-stage-ibm-ssh-key"
   host.ppc1.concurrency: "4"
+
+  host.ibm-gpu-intel64.address: "10.130.81.14"
+  host.ibm-gpu-intel64.platform: "linux-ibm-gpu/intel64"
```
Comment (stuartwdouglas) [inline components/multi-platform-controller/staging-downstream/host-config.yaml:94]: one minor thing, shouldn't this be amd64 not intel64?

Comment (bamachrn) [reply components/multi-platform-controller/staging-downstream/host-config.yaml:94]: its intel machine not AMD so I thought lets make it intel64.

Comment (stuartwdouglas) [reply components/multi-platform-controller/staging-downstream/host-config.yaml:94]: The architecture is called amd64 though

Comment (bamachrn) [reply components/multi-platform-controller/staging-downstream/host-config.yaml:94]: yes, let me change it

**Pull request merged**